src/content/about/blog/oscal-first-deep-cleaning.md

```diff
@@ -0,0 +1,24 @@
+---
+title: OSCAL's Fall Cleaning
+---
+
+Welcome to the Open Security Controls Assessment Language (OSCAL) Blog, open to the NIST OSCAL Team and to the community! 
+
+If you work closely with us, you might have noticed we finished [a large code repository reorganization](https://github.com/usnistgov/OSCAL/blob/f24dd56d5569ade8489924cf6fc2640dc297bfbe/decisions/0005-repository-reorganization.md) alongside [the release of OSCAL v1.1.0](https://github.com/usnistgov/OSCAL/releases/tag/v1.1.0). The OSCAL Project was in a desperate need of some internal restructuring, a fall cleaning of sorts. As an OSCAL user, the differences should not affect affect you, but we will still summarize some key changes for you. 
+
+One may wonder why we reorganized our code repos only recently. Our answer is simple: we started small (more than six years ago!) and wanted to keep everything in one place to reduce the maintenance overhead. As we expanded our work, we needed to separate code repos to get better organized.
+
+Here is what changed:
+
+- The OSCAL website's source code (known as OSCAL pages) moved to its own repo, [OSCAL-Pages](https://github.com/usnistgov/OSCAL-Pages).
+- The OSCAL reference documentation's source code moved to its own repo, [OSCAL-Reference](https://github.com/usnistgov/OSCAL-Reference).
+- The Metaschema tooling's source code, which we use to process OSCAL models and generate documentation, moved to its own repo, [metaschema-xslt](https://github.com/usnistgov/metaschema-xslt).
+- The generated JSON and XML schemas for the models and converters for JSON-XML and XML-JSON conversion are only [published as part of releases](https://github.com/usnistgov/OSCAL/releases), they are not saved in the [OSCAL repo](https://github.com/usnistgov/OSCAL/) like source code anymore. We've also [simplified the build process](https://github.com/usnistgov/OSCAL/blob/main/build/README.md) making it easier for community members to prototype and propose changes to the OSCAL models.
+
+The NIST OSCAL Team officially maintains those new repos above and the same [oscal-content](https://github.com/usnistgov/oscal-content/), [liboscal-java](https://github.com/usnistgov/liboscal-java/), [oscal-cli](https://github.com/usnistgov/oscal-cli/) repos just like before. As priorities shifted, we have paused development on [oscal-cat, the catalog authoring tool](https://github.com/usnistgov/oscal-cat), and [oscal-tools, a library of XSLT transformations for OSCAL developer use cases](https://github.com/usnistgov/oscal-tools). Those projects are archived and remain available for reference and demonstration.
```

Comment:
The sentence could be more concise... Suggestion:

 "The NIST OSCAL team officially maintains those new repos in addition to the existing [OSCAL](https://github.com/usnistgov/OSCAL), [oscal-content](https://github.com/usnistgov/oscal-content/), [liboscal-java](https://github.com/usnistgov/liboscal-java/), and [oscal-cli](https://github.com/usnistgov/oscal-cli/) repos."

Please note that I added OSCAL repo since we are listing all existing repos, I added a conjunction and before the last item in the list, and I used lower case in team.

Reply:
https://github.com/usnistgov/OSCAL-Pages/pull/44/commits/3124d21f0675662f183525a1720957b9d2ce9daf